app.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+if (!process.env.token) {
+  console.log('Error: Specify token in environment');
+  process.exit(1);
+}
+
+
+var Botkit = require('botkit')
+var controller = Botkit.slackbot({
+    debug: false
+});
+
+var bot = controller.spawn({
+    token: process.env.token
```

Comment:
what token is this? real user token? or basic slack bot token? or slack app token?